src/dbnode/storage/namespace.go

```diff
@@ -1693,6 +1694,7 @@ func (n *dbNamespace) aggregateTiles(
 			ctx, sourceNs.ID(), sourceShard.ID(), blockReaders, sourceBlockVolumes, opts, nsCtx.Schema)
 
 		processedTileCount += shardProcessedTileCount
+		processedShards.Inc(1)
```

Comment:
Should this always increment? Probably better to break this out into success/error metrics? Also might be useful to add a metric for `attemptedShards` that would increment at the start of this loop to catch scenarios where we can't read the shard for whatever reason

Reply:
Actually, if the shard fails, the whole job will fail. So kind of failed job count = failed shard count w/o any additional information. Does it really make sense to duplicate it?